obspy/signal/spectral_estimation.py

```diff
@@ -46,8 +46,12 @@
 
 dtiny = np.finfo(0.0).tiny
 
-NOISE_MODEL_FILE = os.path.join(os.path.dirname(__file__),
-                                "data", "noise_models.npz")
+NOISE_MODEL_FILE = os.path.join(os.path.dirname(__file__), "data",
+                                "noise_models.npz")
+
+# Noise models for special_handling="infrasound"
+NOISE_MODEL_FILE_INF = os.path.join(os.path.dirname(__file__), "data",
+                                    "idc_noise_models.npz")
```

Comment:
just a comment, since master is now Python >=3.6 we could use pathlib for stuff like that getting added which is less ugly

```python
Path(__file__).parent / "data" / "idc_noise_models.npz"
```